### PR TITLE
Lower KRB ticket lifetime in EnvMultinodeTlsKerberosDelegation

### PR DESCRIPTION
**Pull request by hashhar**
In 36b7553f219cb47b3abbcd452101ba0c3db6d0b4 the HadoopKerberos environment's ticket lifetime was lowered to be able to test Keberos constrained delegation support in the JDBC driver. The HadoopKerberos environment is a base environment and multiple environments inherit from it. Changing the Kerberos config should be done in dedicated environments to make sure we don't indicentally affect other envs or tests which depend on HadoopKerberos environment.

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HadoopKerberosConstrainedDelegation.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import io.trino.tests.product.launcher.docker.DockerFiles;
+import io.trino.tests.product.launcher.env.Environment;
+
+import java.util.List;
+
+import static io.trino.tests.product.launcher.env.EnvironmentContainers.TESTS;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.utility.MountableFile.forHostPath;
+
+public class HadoopKerberosConstrainedDelegation
```
Comment (nineinchnick) [inline testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HadoopKerberosConstrainedDelegation.java:27]: If this is used in one other env, why add it as a separate class?

Comment (hashhar) [reply testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HadoopKerberosConstrainedDelegation.java:27]: That's a good point as well.

Comment (hashhar) [reply testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HadoopKerberosConstrainedDelegation.java:27]: I wish you were on the review for the original PR which introduced this - so simpler 😄 

Will push the change. Thanks a lot.

**Review by Praveen2112 — APPROVED** [review]
Can we update the PR title and commit message as we are not extracting them

**Comment by hashhar** [comment]
I had a discussion offline with Praveen and we think that we should enable lower lifetime within the Docker images themselves.
This would help find issues with components that don't work with renewable tickets and catch issues like https://github.com/trinodb/trino/pull/16680 earlier.

If we notice any PT env which doesn't work with renewable tickets we can either:

- disable renewable tickets for that PT env
- follow up with investigating how to fix that if possible

**Comment by hashhar** [comment]
@Praveen2112 @nineinchnick addressed comments, PTAL again.

**Review by nineinchnick — APPROVED** [review]

**Pull request merged**